python/tvm/relay/backend/contrib/ethosu/codegen.py

```diff
@@ -361,9 +405,12 @@ def relay_to_tir(mod: tvm.ir.IRModule) -> tvm.ir.IRModule:
     # this should be a single intelligent and a composite scheduler
     # that can perform scheduling based on user inputs such as
     # scratch memory size.
-    mod = LowerToTIR(copy_constants)(mod)
+    if util.enable_cascader() and util.get_accelerator_config() != "ethos-u65-256":
```

Comment:
We should error out if u65 is not supported yet but if its still given as an input

Reply:
Done!